### ISSUE TITLE
[REQ] Show what flags gets added from presets if the flag editor is enabled

### ISSUE THREAD

**Issue by ShadowCodeX-debug** [pizzaboxer/bloxstrap#2221]
### Acknowledgement of preliminary instructions

- [X] I have read and acknowledged the preliminary instructions.

### What idea do you have?

A tooltip that appears after one second of hovering over the preset switch that shows the FFlag that will be added. These tooltips would only show if the editor warning is gone and the editor is accessible.

**Comment by pizzaboxer** [comment]
What's wrong with the toggle button?

**Comment by ShadowCodeX-debug** [comment]
@pizzaboxer I don't think you understood the request.

**Comment by pizzaboxer** [comment]
Yeah, I understand what you mean now. I'll look to see if that's possible.
